packages/arb-token-bridge-ui/src/util/withdrawals/__tests__/fetchTokenWithdrawalsFromEventLogs.test.ts

```diff
@@ -10,21 +10,21 @@ describe('fetchTokenWithdrawalsFromEventLogs', () => {
     // TODO: This is a temporary fix, when Event Logs are enabled for custom address
     // we will be able to use the same properties, and remove the need to assign sender to address
     const query = getQueryCoveringClassicOnlyWithoutResults()
-    const address = query.sender
+    const fromAddress = query.sender
```

Comment:
We don't need  to extract `fromAddress = query.sender` separately here. I don't see `sender` field being used anywhere - Can we  rename `sender` to `fromAddress` in the baseQuery itself?

Reply:
No, because the subgraph tests rely on that, too.